cms/models.py

```diff
@@ -38,7 +38,7 @@ class CourseProgramChildPage(Page):
     class Meta:
         abstract = True
 
-    parent_page_types = ["CoursePage", "ProgramPage"]
+    parent_page_types = ["CoursePage", "ProgramPage", "HomePage"]
 
     # disable promote panels, no need for slug entry, it will be autogenerated
     promote_panels = []
```

Comment:
@ahmed-arbisoft Eventually we have to enable these `promote_panels` in order to add metadata. See for details https://github.com/mitodl/mitxpro/issues/270. Can we enable these fields here ?

Reply:
@tasawernawaz I'd like to keep that separate from this PR as it entails enabling slug fields which are on autogen right now. Also, I don't quite understand the requirement of using `promote` on child pages, seeing as we aren't rendering them as a page in the strictest sense.

Reply:
We don't need this on child pages, we need only on product pages and home page. I will take care of it once this PR is merged.